.travis.yml

```
@@ -1,6 +1,7 @@
 language: node_js
 
 node_js:
+  - "11"
```

Hello,
You can drop Node.js 11 for Node.js 12.
```suggestion
  - "12"
```